tests/test_build.py

```diff
@@ -281,6 +281,28 @@ def test_sidebars_level2(sphinx_build_factory, file_regression):
     file_regression.check(sidebar.prettify(), extension=".html")
 
 
+def test_sidebars_show_nav_level0_index(sphinx_build_factory, file_regression):
+    """"""
+    confoverrides = {"html_theme_options.show_nav_level": 0}
+    sphinx_build = sphinx_build_factory("sidebars", confoverrides=confoverrides).build()
+
+    # Both the column alignment and the margin should be changed
+    index_html = sphinx_build.html_tree("section1/index.html")
+    sidebar = index_html.select("nav#bd-docs-nav")[0]
+    file_regression.check(sidebar.prettify(), extension=".html")
+
+
+def test_sidebars_show_nav_level0_subsection(sphinx_build_factory, file_regression):
```

Comment:
It's not clear to me what this is testing for compared with the test above - what is `subsection`? How is it different from the previous case?

Reply:
it's just checking if the parent parts and sections are collapsed by default, as it is a child page. Have combined both tests into one.